app/views/rails_db/tables/_data.html.erb

```diff
@@ -15,6 +15,7 @@
   <dd>
     <a href="#" data-options="align: bottom" data-dropdown="drop"><%= fa_icon 'th-large' %> Functions</a>
   </dd>
+  <dd style="float:right"><%= link_to raw("#{fa_icon('cogs')} Settings"), '#', :class => 'settings-link', :table_name => "#{@table.name}" %></dd>
```

Comment:
find what class in Foundation CSS framework used for "right" align
and add "clearfix" after it